charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -56,6 +57,8 @@ data:
           - targets: ["localhost:9090"]
       {{ end }}
 
+      # TODO add a proper config flag option for disabling kubernetes-apiservers scraping instead of using useClusterRoles
```

Comment:
When does this `TODO` need to be done? before 0.37.0 or after?

Reply:
we are currently going to give one flag option to disable clusterrole altogether